src/foam/blob/Blob.js

```diff
@@ -29,8 +55,22 @@ foam.INTERFACE({
           name: 'buffer',
         },
         {
-          name: 'offset',
-          of: 'Long'
+          class: 'Int',
+          name: 'offset'
+        }
+      ]
+    },
+    {
+      name: 'getSize',
```

Comment:
Oh I didn't notice you removed the size property.  If we're going to keep it removed, which is fine, we only need getSize.

Blobs are meant to be immutable, so no need for setSize.

Reply:
@adamvy The reason for removing it was that the getSize & setSize functions weren't being properly generated for ProxyBlob

Reply:
Sure that's fine, let's keep it this way, but let's remove setSize() entirely, size is meant to be read only as blobs are immutable.

Reply:
@adamvy Removed the setSize function